app/middleware/simplified-account/simplified-account-strategy.middleware.test.js

```diff
@@ -0,0 +1,209 @@
+const { expect } = require('chai')
+const path = require('path')
+const proxyquire = require('proxyquire')
+const sinon = require('sinon')
+const User = require('../../models/User.class')
+const userFixtures = require('../../../test/fixtures/user.fixtures')
+const stripeAccountSetupFixture = require('../../../test/fixtures/stripe-account-setup.fixtures')
+
+const A_GATEWAY_EXTERNAL_ID = 'a-gateway-external-id'
+const A_SERVICE_EXTERNAL_ID = 'a-service-external-id'
+
+let req, res, next
+
+const buildUser = (serviceExternalId, gatewayAccountIds) => {
+  return new User(userFixtures.validUserResponse({
+    service_roles: [{
+      service: {
+        external_id: serviceExternalId,
+        gateway_account_ids: gatewayAccountIds
+      }
+    }]
+  }))
+}
+
+const setupSimplifiedAccountStrategyTest = function (options) {
+  const {
+    gatewayAccountID,
+    gatewayAccountExternalId,
+    paymentProvider,
+    serviceExternalId,
+    accountType,
+    errorCode
+  } = options
+
+  req = {
+    params: { serviceExternalId, accountType },
+    user: buildUser(serviceExternalId, [`${gatewayAccountID}`])
```

Comment:
Should `[`${gatewayAccountID}`]` just be `[gatewayAccountID]`? Are you intentionally casting it to a string?

Reply:
It's intentional, that's how Users are set on the request